components/object.js

```diff
@@ -2,48 +2,14 @@
 
 var _ = require('underscore');
 
-var promiseOutput = require('../src/promise-output');
-var componentFactory = require('../src/noflo-component-factory');
+var wrapper = require('../src/javascript-wrapper.js');
```

Comment:
object.js was created so that fbp files could define objects from property values. I don't quite follow your second sentence. Can you provide more of an example of an alternative?